gatherers/gather.js

```diff
@@ -17,9 +17,21 @@
 'use strict';
 
 class Gather {
-  static gather() {
-    throw new Error('Must be overridden');
+
+  constructor() {
+    this.artifact = {};
   }
+
+  setup(options) { }
```

Comment:
you can surround these with /* eslint-disable no-unused-vars */ and /* eslint-enable no-unused-vars */ to quiet eslint. I think it's worth keeping them for documentation purposes